RollGen/Utils.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+
+namespace RollGen
+{
+    public static class Utils
+    {
+        public static T ChangeType<T>(object rawEvaluatedExpression)
```

Comment:
@cidthecoatrack As I said earlier, people could use these utilities, so they shouldn't be private.